testing/kuttl/e2e/root-cert-ownership/05--check-secret.yaml

```diff
@@ -25,7 +26,7 @@ commands:
               sleep 1 # this sleep allows time for garbage collector to delete the secret
               ROOT_SECRET=$(kubectl --namespace="${NAMESPACE}" get --ignore-not-found \
                 secret pgo-root-cacert --output name | wc -l)
-              if [[ "$ROOT_SECRET" == 0 ]]; then
+              if [ "$ROOT_SECRET" = 0 ]; then
```

Comment:
❓ should this be == and not just =?

Reply:
As I understand it, in POSIX shells, the single square bracket block evaluates the contained expression according to its `test` utility. The test utility uses a single equals sign (`=`) as the equality operand for strings (I believe the 0 in this expression is coerced to a string).

https://unix.stackexchange.com/questions/168284/unix-test-when-to-use-eq-vs-vs-in-test-commands

Reply:
Thanks for the explanation, learn something new ever day 😄